### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by trbehera**

### CODE REVIEW

**Review by DukeDavis12 — APPROVED** [review]

**Review by Darshini-Parikh93 — DISMISSED** [review]

**Review by DukeDavis12 — DISMISSED** [review]

File: component-samples/demo/README.md [hunk 1]
```diff
@@ -83,20 +93,20 @@ http_proxy_port: 900
...
+***NOTE***: If you are using pre-production devices or an emulated device then certain debug certificates are required. In such cases, it is recommended that the ondie_cache value be set to the <fdo-pri-src>/protocol-samples/ondiecache directory which contains these debug certificates.
 
-`ondie_cache`: (required if supporting OnDie, optional otherwise) specifies the path to the directory containing the OnDie cert and CRLs.
+`ondie_cache`: (required if supporting OnDie, optional otherwise) Specifies the path to the directory containing the OnDie certificates and CRLs.
 
-`ondie_autoupdate`: (optional, default = false) if "true" then the OnDie certs and CRLs are downloaded from the cloud at start up into the directory specified by ondie_cache.
+`ondie_autoupdate`: (optional, default = false) if "true" then the OnDie certificates and CRLs are downloaded from the cloud at start up into the directory specified by ondie_cache.
```
Comment (DukeDavis12) [inline component-samples/demo/README.md:106]: downloaded from the cloud at start up into the directory specified by ondie_cache. -> downloaded from the cloud at start up into the home directory of the docker container.

**Review by DukeDavis12 — DISMISSED** [review]

File: component-samples/demo/README.md (hunk 1 above)
Comment (trbehera) [reply component-samples/demo/README.md:106]: Replaced 'ondie_cache' with home directory in example.

File: component-samples/demo/README.md [hunk 2]
```diff
@@ -110,6 +110,15 @@ Note that this requires internet access by the component. Should a component be
 
 `ondie_check_revocations`: (optional, default = true for Manufacturer and Owner, false for RV) if "true" then revocations are checked by the component, no revocation checking is done if "false".
 
+To enable OnDie support in FDO PRI Manufacturer or FDO PRI Owner, update manufacturer.env or owner.env respectively as below.
```
Comment (DukeDavis12) [inline component-samples/demo/README.md:113]: FDO PRI Manufacturer or FDO PRI Owner, update manufacturer.env or owner.env  -->

FDO PRI Manufacturer and FDO PRI Owner, update manufacturer.env and owner.env.

Comment (trbehera) [reply component-samples/demo/README.md]: Updated in 580a86295bf08d367b91f37c221e122ca651160d

**Review by DukeDavis12 — APPROVED** [review]

**Review by Darshini-Parikh93 — APPROVED** [review]

**Pull request merged**